components/channel_header/__snapshots__/channel_header.test.jsx.snap

```diff
@@ -651,14 +651,13 @@ exports[`components/ChannelHeader should render properly when populated with cha
           defaultOverlayShown={false}
           overlay={
             <Popover
-              bSize="large"
-              bStyle="info"
-              bsClass="popover"
               className="channel-header__popover"
               id="header-popover"
               onMouseOut={[Function]}
               onMouseOver={[Function]}
               placement="bottom"
+              popoverSize="large"
```

Comment:
why the `popover` prefix in the `Popover` component?

Reply:
@jespino - because React's typings already include 'size' and 'style' props for all components. Thus - a different name